assets/_scss/elements/_buttons.scss

```diff
@@ -0,0 +1,77 @@
+// Buttons
+
+input[type="button"],
+input[type="submit"],
+input[type="reset"],
+input[type="image"],
+button[type="button"],
+button[type="submit"],
+button[type="reset"] {
+  -webkit-appearance: none;
+  background-color: $color-grey-900;
+  border: 0;
+  border-radius: $usa-border-radius;
+  color: $color-white;
+  font-size: $base-font-size;
+  font-weight: $font-bold;  
+  line-height: 1;
+  text-align: center;
+  margin: {
+    top: .5em;
+    bottom: .5em;
+    right: .5em;
+  }
+  outline: none;
+  padding: 1rem 2rem;
+  -webkit-font-smoothing: antialiased;  
+
+  &:hover {
+    background-color: $color-grey-700;
+  }
+
+  &:focus {
+    box-shadow: 0 0 0 2px $color-focus;
+  }  
+}
+
+input[type="submit"]:disabled {
+  background-color: $color-grey-300;
+  color: $color-grey-700;
+}
+
+input[type="button"].button-secondary,
+input[type="submit"].button-secondary,
+input[type="reset"].button-secondary,
+input[type="image"].button-secondary,
+button[type="button"].button-secondary,
+button[type="submit"].button-secondary,
+button[type="reset"].button-secondary {
+  background-color: $color-grey-600;
+  
+  &:hover {
+    background-color: $color-grey-800;
+  }  
+}
+
+input[type="button"].button-outlined,
+input[type="submit"].button-outlined,
+input[type="reset"].button-outlined,
+input[type="image"].button-outlined,
+button[type="button"].button-outlined,
+button[type="submit"].button-outlined,
+button[type="reset"].button-outlined {
+  background-color: $color-white;
+  color: $color-grey-900;
+  border: 2px solid $color-grey-900;
+}
+
+input[type="button"].button-big,
+input[type="submit"].button-big,
+input[type="reset"].button-big,
+input[type="image"].button-big,
+button[type="button"].button-big,
+button[type="submit"].button-big,
+button[type="reset"].button-big {
+  font-size: 1.9rem;
+  padding: 1.5rem 3rem;
```

Comment:
From your research, do you think people will want to control button size? We could potentially variablize this sizing stuff.